SupremacyBenchmark/privateBuild/parseLog.py

```diff
@@ -0,0 +1,50 @@
+import re   
```

Comment:
Probably want a comment that this program will take the output of the tests and generate a summary for analysis as an excel spreadsheet

Reply:
I added this as a ToDo on the PR to add a readme with guidance for using these benchmark projects and scripts.

Reply:
Good idea.